src/Donations/Components/DonationsForm.tsx

```diff
@@ -156,6 +156,7 @@ function DonationsForm(): ReactElement {
         setshowErrorCard,
         frequency,
         tenant,
+        locale: i18n.language,
```

Comment:
I wonder if it's possible to avoid this type warning?

<img width="597" alt="Bildschirm­foto 2023-03-18 um 12 48 51" src="https://user-images.githubusercontent.com/1532418/226103739-a73b9a77-5d12-4a4c-a8d7-44d6ddc08fe6.png">

Reply:
This occurred because of the `i18n.d.ts` file I've added at the root of the project with the following code

```
import "i18next";
declare module "i18next" {
  interface CustomTypeOptions {
    returnNull: false;
  }
}
```

Looking for a solution, otherwise I'll revert this change.

Reply:
Fixed in bc40a59697353176ebc7348e85dd1679642f06a2 by moving `i18n.d.ts` into `/@types`